Dockerfile

```diff
@@ -22,4 +16,6 @@ WORKDIR /opt/representer
 COPY --from=build /opt/representer/ .
 COPY --from=build /usr/local/bin/ /usr/local/bin/
 
+COPY generate.sh /opt/representer/bin/
```

Comment:
This should be `run.sh`, not `generate.sh`

Reply:
I know. Will fix it in a follow-up PR